include/invert_quda.h

```diff
@@ -546,7 +543,12 @@ namespace quda {
        @param[in] residual_type The type of residual we want to solve for
        @return L2 stopping condition
     */
-    static double stopping(double tol, double b2, QudaResidualType residual_type);
+    static vector<double> stopping(double tol, cvector<double> &b2, QudaResidualType residual_type);
+
+    static inline double stopping(double tol, double b2, QudaResidualType residual_type)
+    {
+      return stopping(tol, cvector<double>(b2), residual_type)[0];
+    }
 
     /**
        @briefTest for solver convergence
```

Comment:
Randomly noticed this: `@briefTest` -> `@brief Test`

Reply:
f0d9f3c8dc79bdb4ed751b34193262c517a6b287